internal-server.md

```diff
@@ -239,6 +239,15 @@ git push origin :xy/branchname
 git push origin :synced/xy/branchname
 ```
 
+### Updating
+
+To sync up with the latest dataset after a pull request is merged,
+
+```
+git checkout master       
+git pull                  # update metadata files
+git annex get .           # update image files
```

Comment:
will this get _all_ the images? eg with sct-testing-large, that could be a problem. maybe we want to clarify if it downloads everything